### PR TITLE
To Do list: list structure

### PR DESCRIPTION
**Pull request by Paskab012**
## To-Do list: list structure ♻️🥳
While working on this project, I followed instructions given in the manual, and I have implemented the below actions :

- Set up a new project with webpack that is based on the webpack exercise you have already completed.
- Create an index.html file and write your HTML markup here. Create an empty To Do List placeholder (div or ul element). The index.html file must be set as a template using the HTML Webpack Plugin.
- Create an index.js file and set an array of some simple to do tasks (array of objects). 
- Write a function to iterate over the tasks array and populate an HTML list item element for each task.
- Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader. Your list should be a clone of the part of the minimalist project captured in the video below.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
+import './style.css';
+
+let tasks = [];
```
Comment (medaminedev66) [inline src/index.js]: - [ ] Following the project requirements, your project should have an array of some simple to-do tasks (array of objects) with each object containing three keys - description [string], completed [bool], index: [number]. You can create more properties or name these properties differently. Checkout the project requirement bellow 👇 :
image

**Review by medaminedev66 — CHANGES_REQUESTED** [review]
Hi @KABIKA681 , :wave:

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- ✔️ Webpack dev server is set up and running with no errors.

## Required Changes ♻️

_See the inline comments below for more required changes_

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (moise10r) [reply src/index.js]: All good now. Well done!

**Review by moise10r — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @KABIKA681** , 

### **Congratulations! 🎉 Project Approved ✅**
Great work on making the changes requested by the previous reviewer :clap: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @moise10r when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**